cookbook-1.md

```diff
@@ -10,67 +10,142 @@ As Service Consumer, call the API of a Service Provider without being pre-author
 
 ### Get token
```

Comment:
Hier gebeurt denk ik veel te veel: als je als consumer direct een service provider API aan roept zonder delegation evidence mee te geven, dan hoef je niks te doen met het association registry, en ook niks met het authorization registry. Je vraagt dan direct een access token bij de service provider (dus niet bij het assocation registry) en gebruikt die om de uiteindelijke service call te maken

Ik heb hieronder een paar opmerkingen die voor al deze cookbooks gelden, dus let op als je hier stukken weghaalt dat die wel meegenomen worden

Reply:
Waarom? Je moet toch opvragen of de Service Provider nog wel adherent is?